### ISSUE TITLE
Some language files ( zh-TW ) in langs are not auto-sync from transifex

### ISSUE THREAD

**Issue by trekmorris** [Dolibarr/dolibarr#18166]
In github , you could find some lang files in zh-TW that sync more than one month .

**Comment by ksar-ksar** [comment]
Hello,

Transifex has been sync 9 days ago : 
![image](https://user-images.githubusercontent.com/35605507/126288756-3e2646e6-37af-4423-a747-682f857f9a1b.png)

**Comment by trekmorris** [comment]
Hi

 Below are some langs are not sync .

![image](https://user-images.githubusercontent.com/78349579/126337686-d28b4b91-b0bb-4da5-88d3-ecd540ad7c87.png)
![image](https://user-images.githubusercontent.com/78349579/126337732-4b974042-865c-4547-97f7-d215fcdbb3fb.png)
![image](https://user-images.githubusercontent.com/78349579/126337932-dcda35ee-11d4-40bd-822b-79705a8efa30.png)
![image](https://user-images.githubusercontent.com/78349579/126337991-0431617d-38a3-46e9-a09c-f0714be94b73.png)
![image](https://user-images.githubusercontent.com/78349579/126338246-53e46a0f-62b2-4583-92e0-2c4b6279ae0e.png)
![image](https://user-images.githubusercontent.com/78349579/126338295-aa7c3ff4-e429-41de-9308-8a1e2c888c88.png)
![image](https://user-images.githubusercontent.com/78349579/126338374-97ac5d32-09d7-492b-861d-c4e246a85349.png)
![image](https://user-images.githubusercontent.com/78349579/126338418-8f1af5d5-4590-4195-adf2-c7f41129af31.png)
![image](https://user-images.githubusercontent.com/78349579/126339007-ed9a53eb-aeac-4758-a164-5373d7afa7f3.png)
![image](https://user-images.githubusercontent.com/78349579/126339044-868b33c7-83e4-4cd0-9807-6710cfbbae93.png)

**Comment by ksar-ksar** [comment]
Ok then I understand it is the opposite, you think that transifex files are not updated ?
Normally it is not possible....

LDAP last update is 2 august 2020 : 
![image](https://user-images.githubusercontent.com/35605507/126343557-64fbb62a-6e11-4613-ad71-76d2fd95b376.png)

Same for opensurvey
![image](https://user-images.githubusercontent.com/35605507/126343739-3f735ee0-a47c-4df8-9319-6ea0269de07e.png)

etc.

No bug fo rme

**Comment by trekmorris** [comment]
Hi 

Yes , i think some of langs are not sync , because when i open opensurvey.lang of develop , v14 and v13 dolibarr on github .
Then you could compare same opensurvey on transifex , they are different .
I have translated opensurvey  100% on transifex five months ago , but in github it still in english .
And last commit is on 22 Dec 2019 .

![image](https://user-images.githubusercontent.com/78349579/126352260-fc629f49-6ad6-494f-8865-ddae84ac626a.png)
![image](https://user-images.githubusercontent.com/78349579/126352441-c4022be3-2d4d-4bea-b122-4fddd1f62224.png)

**Comment by cfoellmann** [comment]
Is this still an issue?

**Comment by trekmorris** [comment]
Hi 

I am not sure for now , but i have update some translation on transifex  today .
May need to wait 2 weeks to see if it works fine or not .

**Comment by trekmorris** [comment]
After two weeks later , i check the upgrade of translation , it still not upgrade the last translations .
So i think it is still an issue .

**Comment by priojk** [comment]
I can confirm that this issue still exists, in this case for the **de_DE** translations.

Here is one arbitrary example, which may perhaps help to track the source of this problem down:

The key **ProjectFollowTasks** (in projects) with the English standard translation **Follow tasks or time spent**  is in the latest Dolibarr (freshly installed from github) still translated to German as **Befolgen Sie die Aufgaben oder die aufgewendete Zeit**.

I changed this translation in Transifex already 2 months(!) ago to **Aufgaben und aufgewendete Zeiten nachverfolgen**. So several sync cycles did apparently not pick this up.

This is just one example, I remember there were more, which I just noticed but did not write down.
